BraveRewardsUI/Wallet/RewardsSummaryProtocol.swift

```diff
@@ -93,7 +94,10 @@ extension RewardsSummaryProtocol {
       })
     }
     
-    let reservedAmount = BATValue(state.ledger.reservedAmount)
+    state.ledger.pendingContributionsTotal { amount in
+      
```

Comment:
why this empty?

Reply:
Whoops, left this in here when I moved the logic for getting it out (now passed in the arg since its async)

I'll delete it :)